### PR TITLE
A role for PCP

### PR DESCRIPTION
**Pull request by zmc**
This is still a work in progress.

Lots of information on PCP is available here:
http://pcp.io/docs/guide.html

### CODE REVIEW

**Comment by zmc** [comment]
I ran this against ``vpm065`` and it was immediately discoverable by my manually-installed ``pcp`` on ``teuthology.front``:

```
zack@teuthology:~$ pmstat -t 2sec -h vpm065
@ Fri Mar 18 00:27:02 2016
 loadavg                      memory      swap        io    system         cpu
   1 min   swpd   free   buff  cache   pi   po   bi   bo   in   cs  us  sy  id
    0.00      0 233568  76336  1393m    0    0    0    0   19   25   0   0 100
    0.00      0 233476  76336  1393m    0    0    0   24   25  138   0   0  99
    0.00      0 233476  76336  1393m    0    0    0   12   23   27   0   0 100
^C
```

**Comment by zmc** [comment]
And the reverse:
```
ubuntu@vpm065:~$ pmstat -t 2sec -h teuthology
@ Thu Mar 17 17:29:47 2016
 loadavg                      memory      swap        io    system         cpu
   1 min   swpd   free   buff  cache   pi   po   bi   bo   in   cs  us  sy  id
    1.61      0 10247m  85856 10373m    0    0    0    0 4863  12K  13   3  84
    1.48      0 10249m  85860 10374m    0    0    0   56 5238  13K  14   3  83
    1.48      0 10273m  85860 10375m    0    0    0    0 5928  19K  18   7  75
^C
```

File: roles/pcp/defaults/apt_systems.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+- name: Install pcp
+  apt:
+    name: "{{ cobbler_package }}"
```
Comment (dmick) [inline roles/pcp/defaults/apt_systems.yml]: cobbler?

Comment (zmc) [inline roles/pcp/defaults/apt_systems.yml]: copypasta :-/

File: roles/pcp/templates/target-host [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# {{ ansible_managed }}
+{{ lab_domain }}
```
Comment (dmick) [inline roles/pcp/templates/target-host]: doesn't seem like this should be here?  pmmgr(1) says target-host contains "one or more lines containing host specs"

**Comment by zmc** [comment]
Changing this back to DNM for now while I make a few changes

**Pull request merged**